bin/i18n/i18n_script_utils.rb

```diff
@@ -10,30 +10,30 @@
 
 CROWDIN_PROJECTS = {
   codeorg: {
-    config_file: File.join(File.dirname(__FILE__), "codeorg_crowdin.yml"),
-    identity_file: File.join(File.dirname(__FILE__), "crowdin_credentials.yml"),
-    etags_json: File.join(File.dirname(__FILE__), "crowdin", "codeorg_etags.json"),
-    files_to_sync_out_json: File.join(File.dirname(__FILE__), "crowdin", "codeorg_files_to_sync_out.json")
+    config_file:            CDO.dir('bin/i18n/codeorg_crowdin.yml'),
+    identity_file:          CDO.dir('bin/i18n/crowdin_credentials.yml'),
+    etags_json:             CDO.dir('bin/i18n/crowdin/codeorg_etags.json'),
+    files_to_sync_out_json: CDO.dir('bin/i18n/crowdin/codeorg_files_to_sync_out.json')
   },
   'codeorg-markdown': {
-    config_file: File.join(File.dirname(__FILE__), "codeorg_markdown_crowdin.yml"),
-    identity_file: File.join(File.dirname(__FILE__), "crowdin_credentials.yml"),
-    etags_json: File.join(File.dirname(__FILE__), "crowdin", "codeorg-markdown_etags.json"),
-    files_to_sync_out_json: File.join(File.dirname(__FILE__), "crowdin", "codeorg-markdown_files_to_sync_out.json")
+    config_file:            CDO.dir('bin/i18n/codeorg_markdown_crowdin.yml'),
```

Comment:
This may not be an issue anymore but in the past, I have encountered problems when providing paths as some systems use the back slash and others the forward slash.
If is is possible to deal this problem, we should use  `CDO.dir(File.join('bin', 'i18n', 'codeorg_markdown_crowdin.yml'))`

Reply:
1. [File.join](https://rubydoc.info/stdlib/core/3.0.0/File.join) returns a path joined with `/` anyway in any platform, so it still won't solve the issue. This change just make the code more easy to read, in my opinion.
2. Did you encounter the issue or is this some know issue on this project?